src/revolve/api.py

```
@@ -0,0 +1,290 @@
+        }
+        elif req.path.endswith('/db'):
+            env_vars = {
+                "DB_NAME": os.environ.get("DB_NAME", ""),
+                "DB_USER": os.environ.get("DB_USER", ""),
+                "DB_PASSWORD": os.environ.get("DB_PASSWORD", ""),
+                "DB_HOST": os.environ.get("DB_HOST", ""),
+                "DB_PORT": os.environ.get("DB_PORT", ""),
+                "DB_TYPE": os.environ.get("DB_TYPE", ""),
+            }
+        resp.status = falcon.HTTP_200
+        resp.media = env_vars
```

Pylint (3.3.7)

[error] Possibly using variable 'env_vars' before assignment

(E0606)